### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by better319** [dani-garcia/vaultwarden#3449] [Q&A]
i can login on the web，but i can not login by the chrome addon
1、ithink the change the docker time is the answer。wrong

![Snipaste_2023-04-14_11-05-35](https://user-images.githubusercontent.com/13842581/231931638-72757833-5ec8-408f-8381-1a8c57afa3a9.png)
![Snipaste_2023-04-14_11-07-09](https://user-images.githubusercontent.com/13842581/231931644-94f3523a-0218-4d14-b6c2-a1abfbb1cdf4.png)

**Comment by better319** [comment]
i use https:\\xxxxxx.synology.me

maybe the ssl cert？  but i can login https:\\xxxxxx.synology.me：1233

**Comment by better319** [comment]
![Snipaste_2023-04-14_11-17-22](https://user-images.githubusercontent.com/13842581/231932770-f0dde234-0af2-4019-9577-b7f0f775873b.png)

**Comment by better319** [comment]
![Snipaste_2023-04-14_11-19-16](https://user-images.githubusercontent.com/13842581/231932971-0d620c77-321f-4cce-a8f7-2866939d6885.png)

**Comment by BlackDex** [comment]
Looks like you need to update your Vaultwarden instances. You are probably using an older version which isn't compatible anymore.

Please update your container.

**Comment by better319** [comment]
> Looks like you need to update your Vaultwarden instances. You are probably using an older version which isn't compatible anymore.
> 
> Please update your container.

2.25.0

![Snipaste_2023-04-14_16-44-20](https://user-images.githubusercontent.com/13842581/231992830-1f6769a8-b7e5-4201-8166-2adfcfbe4351.png)

**Comment by BlackDex** [comment]
Yes, you need to update to the latest version. The version you have is too old.

**Reply by better319** [reply]
1.21.0 is the latest version？

**Reply by better319** [reply]
![Snipaste_2023-04-14_22-10-35](https://user-images.githubusercontent.com/13842581/232068479-fc81d1e7-5c32-434b-ab98-b5a1403e8afe.png)

**Reply by BlackDex** [reply]
1.28.1 is the latest version

**Reply by better319** [reply]
> 1.28.1 is the latest version

THK

**Reply by better319** [reply]
something had happened: i can download latest version which is named 2.25.0.   but   i  cannot download  the version 1.28.1
![Snipaste_2023-04-15_09-59-18](https://user-images.githubusercontent.com/13842581/232177915-59add514-c39a-4738-8863-c3786e794874.png)
![Snipaste_2023-04-15_09-59-43](https://user-images.githubusercontent.com/13842581/232177918-cc8ffbf7-6a7a-4923-ad09-6009808a75b9.png)

**Reply by better319** [reply]
![Snipaste_2023-04-15_10-03-44](https://user-images.githubusercontent.com/13842581/232177946-21eeb75a-8eec-49a9-9e61-a7eb8e020e9e.png)

**Reply by BlackDex** [reply]
I'm not sure why not. Only thing i can think of is an older version of docker or something else which is wrong.
You could try to download it from quay.io or ghcr.io which both should have the same version.
